### DISCUSSION TITLE
Icons not applying to flatpaks when shown on panel

### DISCUSSION THREAD

**Discussion by ezvision** [lxqt/lxqt#2532] [General]
My custom icons are all working normally except on flatpaks. It displays the default program icon on the panel instead of using the custom one I have set.

The custom icons display normally on the desktop, application menu, and quicklaunch. 

Example with the brave icon:

![Screenshot at 2024-02-27 08-22-51](https://github.com/lxqt/lxqt/assets/31146313/42184956-6d45-488e-b9c7-1a40b8ea9a28)

What could cause this? It's the same thing no matter what icons I use. The icons also work fine on other desktop environments like cinnamon, xfce, etc.

**Comment by tsujan** [comment]
In Task Manager settings, there is a check-box saying "Use icons by WindowClass, if available" Toggle it to see if it has any effect.

**Reply by ezvision** [reply]
That fixed it. Thank you for the help.
